src/scrot.c

```diff
@@ -719,7 +719,6 @@ static char *imPrintf(const char *str, struct tm *tm, const char *filenameIM,
 static char *scrotGetWindowName(Window window)
 {
     assert(disp != NULL);
-    assert(window != None);
```

Comment:
I'd keep this.

Reply:
It's a NULL check:
https://gitlab.freedesktop.org/xorg/proto/xorgproto/-/blob/eb28f0378fadd0d143aad7ec16f7b91814faae9a/include/X11/X.h#L115
https://github.com/resurrecting-open-source-projects/scrot/blob/6e2b5221bc25bb073552a2b0707ee99803f1a44f/src/scrot.c#L690

Reply:
`clientWindow` is not a pointer, window IDs are integers.